openmetadata-ui/src/main/resources/ui/src/pages/LineagePage/LineagePage.tsx

```diff
@@ -300,9 +304,9 @@ const LineagePage = () => {
   return (
     <PageContainerV1>
       <PageLayoutV1 className="p-x-lg">
-        <div className="tw-flex tw-flex-col tw-gap-4 h-full">
+        <div className="lineage-page-container">
           <TitleBreadcrumb titleLinks={titleBreadcrumb} />
-          <div className="p-y-sm p-x-lg h-full tw-flex-1 tw-bg-white">
+          <Card className="h-full">
```

Comment:
`Card` will accept title prop. Please check if we can use it. If not why we can't

Reply:
as discussed, with these changes our page styling will be changed and we don't want that 
<img width="1800" alt="image" src="https://user-images.githubusercontent.com/71748675/198983340-90091826-e02a-4cbf-ad2a-dd46f0a32f51.png">